TrafficCapture/replayerPlugins/jsonMessageTransformers/Es2OsPost23/src/main/java/org/opensearch/migrations/transform/Es2OsPost23.java

```diff
@@ -1,9 +1,30 @@
 package org.opensearch.migrations.transform;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
 public class Es2OsPost23 implements IJsonTransformerProvider {
 
+    @Override
     public JsonJMESPathTransformer createTransformer(Object jsonConfig) {
-        String script = "{ \"settings\": settings, \"mappings\": payload.mappings.*.properties | [merge(@)] }";
+        ObjectMapper objectMapper = new ObjectMapper();
+        Map<String, Object> jsonMap = objectMapper.convertValue(jsonConfig, Map.class);
```

Comment:
Take a look at the provider documentation for `createTransformer`...
```
     * @param jsonConfig is a List, Map, String, or null that should be used to configure the
     *                   IJsonTransformer that is being created
```

What are you trying to do with convert?  If you have a map being passed in, this will work.  Anything else will cause it to throw.  You'd be better off to simply cast this to Map<String,Obiect>.